services/webnn/ort/graph_builder_ort.cc

```diff
@@ -457,10 +479,10 @@ void GraphBuilderOrt::AddClampOperation(const mojom::Clamp& clamp) {
           << "[WebNN] Clamp only supports float32 and float16 data type.";
   }
 
-  uint64_t min_id = NewInitializerAsRawData(
+  uint64_t min_id = NewInitializerAsExternalData(
```

Comment:
I know the min/max can be external data or raw data, but which one is better is a question, right?
I will open an issue for discussion.

Reply:
@huningxin left the comment in #52 : _WebNN clamp min/max values are just scalars. Raw data (copying) vs. external data (non-copying) seems not to be much different._